src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -30,65 +31,161 @@
 final class BinaryOperatorSpacesFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
     /**
-     * @var AbstractAlignFixerHelper[]
+     * @internal
      */
-    private $alignFixerHelpers = [];
+    const SINGLE_SPACE = 'single_space';
+
+    /**
+     * @var array
+     */
+    private static $defaultConfiguration = [
+        '=' => self::SINGLE_SPACE,
```

Comment:
put them all in nested key like `operators`, so one could be able to add next option in future,

Also, I would add a `default` (to not have a need to override all operators one by one)
Then,
```php
'default' => self::SINGLE_SPACE,
'operators' => [
    '=' => null,
    '*' => null,
    // ...
],
```

and then, when I want to change config, I update default or particular operators. when I want to resolve, I take first operator, then default if operator is not configured.

Reply:
Interesting idea.
However I'll wait with updating us the current configuration setup has been used since the start of the PR so maybe people like it better, lets hear from others as well :)

Reply:
they will be able to use "current" way via just adding `operators` envelope and don't give a shit about `default` key.
yet, with "current" way it's not possible to further extend the configuration. Do you remember we had rootless configs that we deprecated and replaced with enveloped ones? That's exactly this scenario!